src/app/index.ts

```diff
@@ -93,6 +100,15 @@ module.exports = class TheiaPlugin extends Base {
             (this.options as any).pluginName = answers.name
         }
 
+        if (!(this.options as any).publisher) {
```

Comment:
what happens if here is no publisher ? (for example I only want to create a plugin locally) ? is there a default publisher value for that case ?

Reply:
You can omit publisher ID and it will equal an empty string.

Reply:
should publisher be then only a "parameter" of the generator ?
(like for theia extension where you can specify license, etc) so we don't ask the publisher information at first

it makes the flow to create the plugin as fast as possible.
We can check that publisher is valid only when we want to publish the extension

Reply:
ok so we remove the question but we let "-p" option to give publisher and later we'll implement `*.rc`stuff ?

Reply:
yes, 
I will add a fix